src/errors.js

```diff
@@ -1,6 +1,14 @@
 import { getNetworkName } from './lib/web3-utils'
 import env from './environment'
 
+export const extendError = (name, { defaultMessage }) =>
+  class extends Error {
+    name = name
+    constructor(message = defaultMessage) {
+      super(message)
+    }
+  }
+
 export class DisputeNotFound extends Error {
```

Comment:
Maybe we can slightly extend the `extendError()` options, such that we can also use it for `DisputeNotFound`?

We could support an arbitrary object and then apply `Object.entries(options).forEach((key, value) => this[key] = value)`.

Reply:
could you give me an example about how are you thinking to do this?

Reply:
Nevermind, what I was thinking doesn't work! Trying to plug this extendibility into the constructor isn't worth it.